libs/blocks/editorial-card/editorial-card.css

```diff
@@ -55,6 +55,11 @@
   min-width: initial;
 }
 
+.editorial-card.max-width {
```

Comment:
I'm guessing this comes from a variation added through Sharepoint? The naming might be a bit misleading, but it should be acceptable for the purposes of the showcase page. Could you add a comment that the variant name might need to be updated?

Reply:
Yes it is. Yea I wasn't sure what the best variation name should be for this but I'm open to suggestions to make this more clear. I'll add in that comment for now.